### ISSUE TITLE
[Bug]: img2img and inpainting causing complete freeze

### ISSUE THREAD

**Issue by Kestral5003** [lshqqytiger/stable-diffusion-webui-directml#260]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits

### What happened?

Img2img and inpaint work once or twice, then complete freeze, nothing on console and have to restart the app

### Steps to reproduce the problem

1. open SD
2. generate an image and send to img2img
3. make 1 or 2 edits
4. freezes

### What should have happened?

should have carried on until i was finished editing

### Sysinfo

{
    "Platform": "Windows-10-10.0.19045-SP0",
    "Python": "3.10.10",
    "Version": "1.6.0",
    "Commit": "39e6d26f3b66805677cc3b5c2bc3bd624c347608",
    "Script path": "B:\\StableDiffusionAMD\\stable-diffusion-webui-directml",
    "Data path": "B:\\StableDiffusionAMD\\stable-diffusion-webui-directml",
    "Extensions dir": "B:\\StableDiffusionAMD\\stable-diffusion-webui-directml\\extensions",
    "Checksum": "8e41cc92548fd8887804e910135a980d07bce65267c9a100a51b33f7fcc92152",
    "Commandline": [
        "launch.py",
        "--medvram",
        "--precision",
        "full",
        "--no-half",
        "--no-half-vae",
        "--opt-split-attention",
        "--disable-nan-check",
        "--autolaunch",
        "--api"
    ],
    "Torch env info": {
        "torch_version": "2.0.0+cpu",
        "is_debug_build": "False",
        "cuda_compiled_version": null,
        "gcc_version": null,
        "clang_version": null,
        "cmake_version": null,
        "os": "Microsoft Windows 10 Home",
        "libc_version": "N/A",
        "python_version": "3.10.10 (tags/v3.10.10:aad5f6a, Feb  7 2023, 17:20:36) [MSC v.1929 64 bit (AMD64)] (64-bit runtime)",
        "python_platform": "Windows-10-10.0.19045-SP0",
        "is_cuda_available": "False",
        "cuda_runtime_version": null,
        "cuda_module_loading": "N/A",
        "nvidia_driver_version": null,
        "nvidia_gpu_models": null,
        "cudnn_version": null,
        "pip_version": "pip3",
        "pip_packages": [
            "numpy==1.23.5",
            "open-clip-torch==2.20.0",
            "pytorch-lightning==1.9.4",
            "torch==2.0.0",
            "torch-directml==0.2.0.dev230426",
            "torchdiffeq==0.2.3",
            "torchmetrics==1.1.1",
            "torchsde==0.2.5",
            "torchvision==0.15.1"
        ],
        "conda_packages": null,
        "hip_compiled_version": "N/A",
        "hip_runtime_version": "N/A",
        "miopen_runtime_version": "N/A",
        "caching_allocator_config": "garbage_collection_threshold:0.9,max_split_size_mb:512",
        "is_xnnpack_available": "True",
        "cpu_info": [
            "Architecture=9",
            "CurrentClockSpeed=3912",
            "DeviceID=CPU0",
            "Family=205",
            "L2CacheSize=3072",
            "L2CacheSpeed=",
            "Manufacturer=GenuineIntel",
            "MaxClockSpeed=3912",
            "Name=11th Gen Intel(R) Core(TM) i5-11600K @ 3.90GHz",
            "ProcessorType=3",
            "Revision="
        ]
    },
    "Exceptions": [],
    "CPU": {
        "model": "Intel64 Family 6 Model 167 Stepping 1, GenuineIntel",
        "count logical": 12,
        "count physical": 6
    },
    "RAM": {
        "total": "32GB",
        "used": "18GB",
        "free": "13GB"
    },
    "Extensions": [
        {
            "name": "Stable-Diffusion-Webui-Civitai-Helper",
            "path": "B:\\StableDiffusionAMD\\stable-diffusion-webui-directml\\extensions\\Stable-Diffusion-Webui-Civitai-Helper",
            "version": "920ca326",
            "branch": "main",
            "remote": "https://github.com/butaixianran/Stable-Diffusion-Webui-Civitai-Helper.git"
        },
        {
            "name": "adetailer",
            "path": "B:\\StableDiffusionAMD\\stable-diffusion-webui-directml\\extensions\\adetailer",
            "version": "451b0985",
            "branch": "main",
            "remote": "https://github.com/Bing-su/adetailer.git"
        },
        {
            "name": "sd-civitai-browser-plus",
            "path": "B:\\StableDiffusionAMD\\stable-diffusion-webui-directml\\extensions\\sd-civitai-browser-plus",
            "version": "394c3ea2",
            "branch": "main",
            "remote": "https://github.com/BlafKing/sd-civitai-browser-plus.git"
        }
    ],
    "Inactive extensions": [
        {
            "name": "sd-dynamic-prompts",
            "path": "B:\\StableDiffusionAMD\\stable-diffusion-webui-directml\\extensions\\sd-dynamic-prompts",
            "version": "39c06b30",
            "branch": "main",
            "remote": "https://github.com/adieyal/sd-dynamic-prompts.git"
        }
    ],
    "Environment": {
        "COMMANDLINE_ARGS": "--medvram --precision full --no-half --no-half-vae --opt-split-attention --disable-nan-check --autolaunch --api",
        "GRADIO_ANALYTICS_ENABLED": "False"
    },
    "Config": {
        "samples_save": true,
        "samples_format": "png",
        "samples_filename_pattern": "",
        "save_images_add_number": true,
        "grid_save": true,
        "grid_format": "png",
        "grid_extended_filename": false,
        "grid_only_if_multiple": true,
        "grid_prevent_empty_spots": false,
        "grid_zip_filename_pattern": "",
        "n_rows": -1,
        "font": "",
        "grid_text_active_color": "#000000",
        "grid_text_inactive_color": "#999999",
        "grid_background_color": "#ffffff",
        "enable_pnginfo": true,
        "save_txt": false,
        "save_images_before_face_restoration": false,
        "save_images_before_highres_fix": false,
        "save_images_before_color_correction": false,
        "save_mask": false,
        "save_mask_composite": false,
        "jpeg_quality": 80,
        "webp_lossless": false,
        "export_for_4chan": true,
        "img_downscale_threshold": 4.0,
        "target_side_length": 4000,
        "img_max_size_mp": 200,
        "use_original_name_batch": true,
        "use_upscaler_name_as_suffix": false,
        "save_selected_only": true,
        "save_init_img": false,
        "temp_dir": "",
        "clean_temp_dir_at_start": false,
        "save_incomplete_images": false,
        "outdir_samples": "",
        "outdir_txt2img_samples": "outputs/txt2img-images",
        "outdir_img2img_samples": "outputs/img2img-images",
        "outdir_extras_samples": "outputs/extras-images",
        "outdir_grids": "",
        "outdir_txt2img_grids": "outputs/txt2img-grids",
        "outdir_img2img_grids": "outputs/img2img-grids",
        "outdir_save": "log/images",
        "outdir_init_images": "outputs/init-images",
        "save_to_dirs": true,
        "grid_save_to_dirs": true,
        "use_save_to_dirs_for_ui": false,
        "directories_filename_pattern": "[date]",
        "directories_max_prompt_words": 8,
        "ESRGAN_tile": 192,
        "ESRGAN_tile_overlap": 8,
        "realesrgan_enabled_models": [
            "R-ESRGAN 4x+",
            "R-ESRGAN 4x+ Anime6B"
        ],
        "upscaler_for_img2img": null,
        "face_restoration": false,
        "face_restoration_model": "CodeFormer",
        "code_former_weight": 0.5,
        "face_restoration_unload": false,
        "auto_launch_browser": "Local",
        "show_warnings": false,
        "show_gradio_deprecation_warnings": true,
        "memmon_poll_rate": 8,
        "samples_log_stdout": false,
        "multiple_tqdm": true,
        "print_hypernet_extra": false,
        "list_hidden_files": true,
        "disable_mmap_load_safetensors": false,
        "hide_ldm_prints": true,
        "api_enable_requests": true,
        "api_forbid_local_requests": true,
        "api_useragent": "",
        "unload_models_when_training": false,
        "pin_memory": false,
        "save_optimizer_state": false,
        "save_training_settings_to_txt": true,
        "dataset_filename_word_regex": "",
        "dataset_filename_join_string": " ",
        "training_image_repeats_per_epoch": 1,
        "training_write_csv_every": 500,
        "training_xattention_optimizations": false,
        "training_enable_tensorboard": false,
        "training_tensorboard_save_images": false,
        "training_tensorboard_flush_every": 120,
        "sd_model_checkpoint": "42424\\meinahentai_v4.safetensors [8145104977]",
        "sd_checkpoints_limit": 1,
        "sd_checkpoints_keep_in_cpu": true,
        "sd_checkpoint_cache": 0,
        "sd_unet": "Automatic",
        "enable_quantization": false,
        "enable_emphasis": true,
        "enable_batch_seeds": true,
        "comma_padding_backtrack": 20,
        "CLIP_stop_at_last_layers": 2,
        "upcast_attn": false,
        "randn_source": "GPU",
        "tiling": false,
        "hires_fix_refiner_pass": "second pass",
        "sdxl_crop_top": 0,
        "sdxl_crop_left": 0,
        "sdxl_refiner_low_aesthetic_score": 2.5,
        "sdxl_refiner_high_aesthetic_score": 6.0,
        "sd_vae_explanation": "<abbr title='Variational autoencoder'>VAE</abbr> is a neural network that transforms a standard <abbr title='red/green/blue'>RGB</abbr>\nimage into latent space representation and back. Latent space representation is what stable diffusion is working on during sampling\n(i.e. when the progress bar is between empty and full). For txt2img, VAE is used to create a resulting image after the sampling is finished.\nFor img2img, VAE is used to process user's input image before the sampling, and to create an image after sampling.",
        "sd_vae_checkpoint_cache": 0,
        "sd_vae": "kl-f8-anime2.ckpt",
        "sd_vae_overrides_per_model_preferences": true,
        "auto_vae_precision": true,
        "sd_vae_encode_method": "Full",
        "sd_vae_decode_method": "Full",
        "inpainting_mask_weight": 1.0,
        "initial_noise_multiplier": 1.0,
        "img2img_extra_noise": 0.0,
        "img2img_color_correction": false,
        "img2img_fix_steps": false,
        "img2img_background_color": "#ffffff",
        "img2img_editor_height": 720,
        "img2img_sketch_default_brush_color": "#ffffff",
        "img2img_inpaint_mask_brush_color": "#ffffff",
        "img2img_inpaint_sketch_default_brush_color": "#ffffff",
        "return_mask": false,
        "return_mask_composite": false,
        "enable_mem_pattern": true,
        "enable_mem_reuse": true,
        "reload_model_before_each_generation": false,
        "offload_state_dict": false,
        "cross_attention_optimization": "Automatic",
        "s_min_uncond": 0.0,
        "token_merging_ratio": 0.0,
        "token_merging_ratio_img2img": 0.0,
        "token_merging_ratio_hr": 0.0,
        "pad_cond_uncond": false,
        "persistent_cond_cache": true,
        "batch_cond_uncond": true,
        "directml_memory_provider": "Performance Counter",
        "use_old_emphasis_implementation": false,
        "use_old_karras_scheduler_sigmas": false,
        "no_dpmpp_sde_batch_determinism": false,
        "use_old_hires_fix_width_height": false,
        "dont_fix_second_order_samplers_schedule": false,
        "hires_fix_use_firstpass_conds": false,
        "use_old_scheduling": false,
        "interrogate_keep_models_in_memory": false,
        "interrogate_return_ranks": false,
        "interrogate_clip_num_beams": 1,
        "interrogate_clip_min_length": 24,
        "interrogate_clip_max_length": 48,
        "interrogate_clip_dict_limit": 1500,
        "interrogate_clip_skip_categories": [],
        "interrogate_deepbooru_score_threshold": 0.5,
        "deepbooru_sort_alpha": true,
        "deepbooru_use_spaces": true,
        "deepbooru_escape": true,
        "deepbooru_filter_tags": "",
        "extra_networks_show_hidden_directories": true,
        "extra_networks_hidden_models": "When searched",
        "extra_networks_default_multiplier": 1.0,
        "extra_networks_card_width": 0,
        "extra_networks_card_height": 0,
        "extra_networks_card_text_scale": 1.0,
        "extra_networks_card_show_desc": true,
        "extra_networks_add_text_separator": " ",
        "ui_extra_networks_tab_reorder": "",
        "textual_inversion_print_at_load": false,
        "textual_inversion_add_hashes_to_infotext": true,
        "sd_hypernetwork": "None",
        "localization": "None",
        "gradio_theme": "Default",
        "gradio_themes_cache": true,
        "gallery_height": "",
        "return_grid": true,
        "do_not_show_images": false,
        "send_seed": true,
        "send_size": true,
        "js_modal_lightbox": true,
        "js_modal_lightbox_initially_zoomed": true,
        "js_modal_lightbox_gamepad": false,
        "js_modal_lightbox_gamepad_repeat": 250,
        "show_progress_in_title": true,
        "samplers_in_dropdown": true,
        "dimensions_and_batch_together": true,
        "keyedit_precision_attention": 0.1,
        "keyedit_precision_extra": 0.05,
        "keyedit_delimiters": ".,\\/!?%^*;:{}=`~()",
        "keyedit_move": true,
        "quicksettings_list": [
            "sd_model_checkpoint",
            "sd_vae"
        ],
        "ui_tab_order": [],
        "hidden_tabs": [],
        "ui_reorder_list": [],
        "hires_fix_show_sampler": false,
        "hires_fix_show_prompts": false,
        "disable_token_counters": false,
        "add_model_hash_to_info": true,
        "add_model_name_to_info": true,
        "add_user_name_to_info": false,
        "add_version_to_infotext": true,
        "disable_weights_auto_swap": true,
        "infotext_styles": "Apply if any",
        "show_progressbar": true,
        "live_previews_enable": true,
        "live_previews_image_format": "png",
        "show_progress_grid": true,
        "show_progress_every_n_steps": 2,
        "show_progress_type": "Approx NN",
        "live_preview_allow_lowvram_full": false,
        "live_preview_content": "Combined",
        "live_preview_refresh_period": 150.0,
        "live_preview_fast_interrupt": true,
        "hide_samplers": [],
        "eta_ddim": 0.0,
        "eta_ancestral": 1.0,
        "ddim_discretize": "uniform",
        "s_churn": 0.0,
        "s_tmin": 0.0,
        "s_tmax": 0.0,
        "s_noise": 1.0,
        "k_sched_type": "Automatic",
        "sigma_min": 0.0,
        "sigma_max": 0.0,
        "rho": 0.0,
        "eta_noise_seed_delta": 0,
        "always_discard_next_to_last_sigma": false,
        "sgm_noise_multiplier": false,
        "uni_pc_variant": "bh1",
        "uni_pc_skip_type": "time_uniform",
        "uni_pc_order": 3,
        "uni_pc_lower_order_final": true,
        "postprocessing_enable_in_main_ui": [],
        "postprocessing_operation_order": [],
        "upscaling_max_images_in_cache": 5,
        "disabled_extensions": [
            "sd-dynamic-prompts"
        ],
        "disable_all_extensions": "none",
        "restore_config_state_file": "",
        "sd_checkpoint_hash": "814510497782af7b2ed8b7cfcd9c4011a2ad9e93ddc0ee00b7041a53209ff45a",
        "ldsr_steps": 100,
        "ldsr_cached": false,
        "SCUNET_tile": 256,
        "SCUNET_tile_overlap": 8,
        "SWIN_tile": 192,
        "SWIN_tile_overlap": 8,
        "lora_functional": false,
        "sd_lora": "None",
        "lora_preferred_name": "Alias from file",
        "lora_add_hashes_to_infotext": true,
        "lora_show_all": false,
        "lora_hide_unknown_for_versions": [],
        "lora_in_memory_limit": 0,
        "extra_options_txt2img": [],
        "extra_options_img2img": [],
        "extra_options_cols": 1,
        "extra_options_accordion": false,
        "canvas_hotkey_zoom": "Alt",
        "canvas_hotkey_adjust": "Ctrl",
        "canvas_hotkey_move": "F",
        "canvas_hotkey_fullscreen": "S",
        "canvas_hotkey_reset": "R",
        "canvas_hotkey_overlap": "O",
        "canvas_show_tooltip": true,
        "canvas_auto_expand": true,
        "canvas_blur_prompt": false,
        "canvas_disabled_functions": [
            "Overlap"
        ],
        "ad_max_models": 2,
        "ad_save_previews": false,
        "ad_save_images_before": false,
        "ad_only_seleted_scripts": true,
        "ad_script_names": "dynamic_prompting,dynamic_thresholding,wildcard_recursive,wildcards,lora_block_weight",
        "ad_bbox_sortby": "None"
    },
    "Startup": {
        "total": 188.2950975894928,
        "records": {
            "initial startup": 0.00049591064453125,
            "prepare environment/checks": 0.01735997200012207,
            "prepare environment/git version info": 0.03868699073791504,
            "prepare environment/install torch": 44.06099224090576,
            "prepare environment/install clip": 3.574171543121338,
            "prepare environment/install open_clip": 5.40341854095459,
            "prepare environment/clone repositores": 0.1314406394958496,
            "prepare environment/install CodeFormer requirements": 28.866667985916138,
            "prepare environment/install requirements": 76.29135942459106,
            "prepare environment/run extensions installers/adetailer": 16.969059228897095,
            "prepare environment/run extensions installers/sd-civitai-browser-plus": 0.00046253204345703125,
            "prepare environment/run extensions installers/Stable-Diffusion-Webui-Civitai-Helper": 0.0,
            "prepare environment/run extensions installers": 16.96952176094055,
            "prepare environment": 175.35411548614502,
            "launcher": 0.0014879703521728516,
            "import torch": 4.093292474746704,
            "import gradio": 1.575258493423462,
            "setup paths": 1.5174031257629395,
            "import ldm": 0.004464387893676758,
            "import sgm": 0.0,
            "initialize shared": 1.6665785312652588,
            "other imports": 0.33087682723999023,
            "opts onchange": 0.0,
            "setup SD model": 0.001983165740966797,
            "setup codeformer": 0.014876127243041992,
            "setup gfpgan": 0.015406131744384766,
            "set samplers": 0.0,
            "list extensions": 0.0005288124084472656,
            "restore config state file": 0.0,
            "list SD models": 0.025269031524658203,
            "list localizations": 0.0,
            "load scripts/custom_code.py": 0.0019881725311279297,
            "load scripts/img2imgalt.py": 0.0004990100860595703,
            "load scripts/loopback.py": 0.0,
            "load scripts/outpainting_mk_2.py": 0.0004956722259521484,
            "load scripts/poor_mans_outpainting.py": 0.0004954338073730469,
            "load scripts/postprocessing_codeformer.py": 0.0,
            "load scripts/postprocessing_gfpgan.py": 0.0004968643188476562,
            "load scripts/postprocessing_upscale.py": 0.0,
            "load scripts/prompt_matrix.py": 0.0004956722259521484,
            "load scripts/prompts_from_file.py": 0.0,
            "load scripts/refiner.py": 0.00049591064453125,
            "load scripts/sd_upscale.py": 0.0,
            "load scripts/seed.py": 0.0004956722259521484,
            "load scripts/xyz_grid.py": 0.0009920597076416016,
            "load scripts/civitai_helper.py": 0.6066126823425293,
            "load scripts/!adetailer.py": 1.778160572052002,
            "load scripts/civitai-api.py": 0.04244685173034668,
            "load scripts/ldsr_model.py": 0.027771472930908203,
            "load scripts/lora_script.py": 0.1715693473815918,
            "load scripts/scunet_model.py": 0.02737569808959961,
            "load scripts/swinir_model.py": 0.026291608810424805,
            "load scripts/hotkey_config.py": 0.000492095947265625,
            "load scripts/extra_options_section.py": 0.0004961490631103516,
            "load scripts": 2.687670946121216,
            "load upscalers": 0.0024797916412353516,
            "refresh VAE": 0.0019838809967041016,
            "refresh textual inversion templates": 0.0,
            "scripts list_optimizers": 0.0005002021789550781,
            "scripts list_unets": 0.0,
            "reload hypernetworks": 0.002971172332763672,
            "initialize extra networks": 0.011410236358642578,
            "scripts before_ui_callback": 0.0009908676147460938,
            "create ui": 0.6502549648284912,
            "gradio launch": 0.2599031925201416,
            "add APIs": 0.07489562034606934,
            "app_started_callback/lora_script.py": 0.0004966259002685547,
            "app_started_callback": 0.0004966259002685547
        }
    },
    "Packages": [
        "absl-py==1.4.0",
        "accelerate==0.21.0",
        "addict==2.4.0",
        "aenum==3.1.15",
        "aiofiles==23.2.1",
        "aiohttp==3.8.5",
        "aiosignal==1.3.1",
        "altair==5.1.1",
        "antlr4-python3-runtime==4.9.3",
        "anyio==3.7.1",
        "async-timeout==4.0.3",
        "attrs==23.1.0",
        "basicsr==1.4.2",
        "beautifulsoup4==4.12.2",
        "blendmodes==2022",
        "boltons==23.0.0",
        "cachetools==5.3.1",
        "certifi==2023.7.22",
        "cffi==1.15.1",
        "charset-normalizer==3.2.0",
        "clean-fid==0.1.35",
        "click==8.1.7",
        "clip==1.0",
        "colorama==0.4.6",
        "contourpy==1.1.0",
        "cycler==0.11.0",
        "deprecation==2.1.0",
        "diffusers==0.20.1",
        "einops==0.4.1",
        "exceptiongroup==1.1.3",
        "facexlib==0.3.0",
        "fastapi==0.94.0",
        "ffmpy==0.3.1",
        "filelock==3.12.3",
        "filterpy==1.4.5",
        "flatbuffers==23.5.26",
        "fonttools==4.42.1",
        "frozenlist==1.4.0",
        "fsspec==2023.9.0",
        "ftfy==6.1.1",
        "future==0.18.3",
        "gdown==4.7.1",
        "gfpgan==1.3.8",
        "gitdb==4.0.10",
        "gitpython==3.1.32",
        "google-auth-oauthlib==1.0.0",
        "google-auth==2.22.0",
        "gradio-client==0.5.0",
        "gradio==3.41.2",
        "grpcio==1.57.0",
        "h11==0.12.0",
        "httpcore==0.15.0",
        "httpx==0.24.1",
        "huggingface-hub==0.16.4",
        "idna==3.4",
        "imageio==2.31.2",
        "importlib-metadata==6.8.0",
        "importlib-resources==6.0.1",
        "inflection==0.5.1",
        "jinja2==3.1.2",
        "jsonmerge==1.8.0",
        "jsonschema-specifications==2023.7.1",
        "jsonschema==4.19.0",
        "kiwisolver==1.4.5",
        "kornia==0.6.7",
        "lark==1.1.2",
        "lazy-loader==0.3",
        "lightning-utilities==0.9.0",
        "llvmlite==0.40.1",
        "lmdb==1.4.1",
        "lpips==0.1.4",
        "markdown-it-py==3.0.0",
        "markdown==3.4.4",
        "markupsafe==2.1.3",
        "matplotlib==3.7.2",
        "mdurl==0.1.2",
        "mediapipe==0.10.3",
        "mpmath==1.3.0",
        "multidict==6.0.4",
        "networkx==3.1",
        "numba==0.57.1",
        "numpy==1.23.5",
        "oauthlib==3.2.2",
        "omegaconf==2.2.3",
        "open-clip-torch==2.20.0",
        "opencv-contrib-python==4.8.0.76",
        "opencv-python==4.8.0.76",
        "orjson==3.9.5",
        "packaging==23.1",
        "pandas==2.1.0",
        "piexif==1.1.3",
        "pillow==9.5.0",
        "pip==22.3.1",
        "platformdirs==3.10.0",
        "protobuf==3.20.0",
        "psutil==5.9.5",
        "py-cpuinfo==9.0.0",
        "pyasn1-modules==0.3.0",
        "pyasn1==0.5.0",
        "pycparser==2.21",
        "pydantic==1.10.12",
        "pydub==0.25.1",
        "pygments==2.16.1",
        "pyparsing==3.0.9",
        "pysocks==1.7.1",
        "python-dateutil==2.8.2",
        "python-multipart==0.0.6",
        "pytorch-lightning==1.9.4",
        "pytz==2023.3",
        "pywavelets==1.4.1",
        "pyyaml==6.0.1",
        "realesrgan==0.3.0",
        "referencing==0.30.2",
        "regex==2023.8.8",
        "requests-oauthlib==1.3.1",
        "requests==2.31.0",
        "resize-right==0.0.2",
        "rich==13.5.2",
        "rpds-py==0.10.0",
        "rsa==4.9",
        "safetensors==0.3.1",
        "scikit-image==0.21.0",
        "scipy==1.11.2",
        "seaborn==0.12.2",
        "semantic-version==2.10.0",
        "sentencepiece==0.1.99",
        "setuptools==65.5.0",
        "six==1.16.0",
        "smmap==5.0.0",
        "sniffio==1.3.0",
        "sounddevice==0.4.6",
        "soupsieve==2.5",
        "starlette==0.26.1",
        "sympy==1.12",
        "tb-nightly==2.15.0a20230902",
        "tensorboard-data-server==0.7.1",
        "tifffile==2023.8.30",
        "timm==0.9.2",
        "tokenizers==0.13.3",
        "tomesd==0.1.3",
        "tomli==2.0.1",
        "toolz==0.12.0",
        "torch-directml==0.2.0.dev230426",
        "torch==2.0.0",
        "torchdiffeq==0.2.3",
        "torchmetrics==1.1.1",
        "torchsde==0.2.5",
        "torchvision==0.15.1",
        "tqdm==4.66.1",
        "trampoline==0.1.2",
        "transformers==4.30.2",
        "typing-extensions==4.7.1",
        "tzdata==2023.3",
        "ultralytics==8.0.169",
        "urllib3==1.26.16",
        "uvicorn==0.23.2",
        "wcwidth==0.2.6",
        "websockets==11.0.3",
        "werkzeug==2.3.7",
        "wheel==0.41.2",
        "yapf==0.40.1",
        "yarl==1.9.2",
        "zipp==3.16.2"
    ]
}

### What browsers do you use to access the UI ?

Other

### Console logs

```Shell
Already up to date.
Creating venv in directory B:\StableDiffusionAMD\stable-diffusion-webui-directml\venv using python "C:\Users\Danny\AppData\Local\Programs\Python\Python310\python.exe"
venv "B:\StableDiffusionAMD\stable-diffusion-webui-directml\venv\Scripts\Python.exe"
fatal: No names found, cannot describe anything.
Python 3.10.10 (tags/v3.10.10:aad5f6a, Feb  7 2023, 17:20:36) [MSC v.1929 64 bit (AMD64)]
Version: 1.6.0
Commit hash: 39e6d26f3b66805677cc3b5c2bc3bd624c347608
Installing torch and torchvision
Collecting torch==2.0.0
  Using cached torch-2.0.0-cp310-cp310-win_amd64.whl (172.3 MB)
Collecting torchvision==0.15.1
  Using cached torchvision-0.15.1-cp310-cp310-win_amd64.whl (1.2 MB)
Collecting torch-directml
  Using cached torch_directml-0.2.0.dev230426-cp310-cp310-win_amd64.whl (8.2 MB)
Collecting typing-extensions
  Using cached typing_extensions-4.7.1-py3-none-any.whl (33 kB)
Collecting filelock
  Using cached filelock-3.12.3-py3-none-any.whl (11 kB)
Collecting networkx
  Using cached networkx-3.1-py3-none-any.whl (2.1 MB)
Collecting jinja2
  Using cached Jinja2-3.1.2-py3-none-any.whl (133 kB)
Collecting sympy
  Using cached sympy-1.12-py3-none-any.whl (5.7 MB)
Collecting numpy
  Using cached numpy-1.25.2-cp310-cp310-win_amd64.whl (15.6 MB)
Collecting pillow!=8.3.*,>=5.3.0
  Using cached Pillow-10.0.0-cp310-cp310-win_amd64.whl (2.5 MB)
Collecting requests
  Using cached requests-2.31.0-py3-none-any.whl (62 kB)
Collecting MarkupSafe>=2.0
  Using cached MarkupSafe-2.1.3-cp310-cp310-win_amd64.whl (17 kB)
Collecting urllib3<3,>=1.21.1
  Using cached urllib3-2.0.4-py3-none-any.whl (123 kB)
Collecting certifi>=2017.4.17
  Using cached certifi-2023.7.22-py3-none-any.whl (158 kB)
Collecting idna<4,>=2.5
  Using cached idna-3.4-py3-none-any.whl (61 kB)
Collecting charset-normalizer<4,>=2
  Using cached charset_normalizer-3.2.0-cp310-cp310-win_amd64.whl (96 kB)
Collecting mpmath>=0.19
  Using cached mpmath-1.3.0-py3-none-any.whl (536 kB)
Installing collected packages: mpmath, urllib3, typing-extensions, sympy, pillow, numpy, networkx, MarkupSafe, idna, charset-normalizer, certifi, requests, jinja2, filelock, torch, torchvision, torch-directml
Successfully installed MarkupSafe-2.1.3 certifi-2023.7.22 charset-normalizer-3.2.0 filelock-3.12.3 idna-3.4 jinja2-3.1.2 mpmath-1.3.0 networkx-3.1 numpy-1.25.2 pillow-10.0.0 requests-2.31.0 sympy-1.12 torch-2.0.0 torch-directml-0.2.0.dev230426 torchvision-0.15.1 typing-extensions-4.7.1 urllib3-2.0.4

[notice] A new release of pip available: 22.3.1 -> 23.2.1
[notice] To update, run: B:\StableDiffusionAMD\stable-diffusion-webui-directml\venv\Scripts\python.exe -m pip install --upgrade pip
Installing clip
Installing open_clip
Installing requirements for CodeFormer
Installing requirements
Collecting ultralytics>=8.0.167
  Using cached ultralytics-8.0.169-py3-none-any.whl (613 kB)
Requirement already satisfied: requests>=2.23.0 in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from ultralytics>=8.0.167) (2.31.0)
Requirement already satisfied: scipy>=1.4.1 in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from ultralytics>=8.0.167) (1.11.2)
Requirement already satisfied: psutil in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from ultralytics>=8.0.167) (5.9.5)
Collecting py-cpuinfo
  Using cached py_cpuinfo-9.0.0-py3-none-any.whl (22 kB)
Requirement already satisfied: opencv-python>=4.6.0 in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from ultralytics>=8.0.167) (4.8.0.76)
Requirement already satisfied: pandas>=1.1.4 in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from ultralytics>=8.0.167) (2.1.0)
Requirement already satisfied: pyyaml>=5.3.1 in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from ultralytics>=8.0.167) (6.0.1)
Requirement already satisfied: torchvision>=0.9.0 in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from ultralytics>=8.0.167) (0.15.1)
Requirement already satisfied: torch>=1.8.0 in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from ultralytics>=8.0.167) (2.0.0)
Requirement already satisfied: numpy>=1.22.2 in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from ultralytics>=8.0.167) (1.23.5)
Requirement already satisfied: pillow>=7.1.2 in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from ultralytics>=8.0.167) (9.5.0)
Requirement already satisfied: matplotlib>=3.2.2 in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from ultralytics>=8.0.167) (3.7.2)
Requirement already satisfied: tqdm>=4.64.0 in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from ultralytics>=8.0.167) (4.66.1)
Collecting seaborn>=0.11.0
  Using cached seaborn-0.12.2-py3-none-any.whl (293 kB)
Requirement already satisfied: python-dateutil>=2.7 in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from matplotlib>=3.2.2->ultralytics>=8.0.167) (2.8.2)
Requirement already satisfied: packaging>=20.0 in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from matplotlib>=3.2.2->ultralytics>=8.0.167) (23.1)
Requirement already satisfied: cycler>=0.10 in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from matplotlib>=3.2.2->ultralytics>=8.0.167) (0.11.0)
Requirement already satisfied: pyparsing<3.1,>=2.3.1 in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from matplotlib>=3.2.2->ultralytics>=8.0.167) (3.0.9)
Requirement already satisfied: kiwisolver>=1.0.1 in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from matplotlib>=3.2.2->ultralytics>=8.0.167) (1.4.5)
Requirement already satisfied: contourpy>=1.0.1 in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from matplotlib>=3.2.2->ultralytics>=8.0.167) (1.1.0)
Requirement already satisfied: fonttools>=4.22.0 in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from matplotlib>=3.2.2->ultralytics>=8.0.167) (4.42.1)
Requirement already satisfied: tzdata>=2022.1 in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from pandas>=1.1.4->ultralytics>=8.0.167) (2023.3)
Requirement already satisfied: pytz>=2020.1 in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from pandas>=1.1.4->ultralytics>=8.0.167) (2023.3)
Requirement already satisfied: idna<4,>=2.5 in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from requests>=2.23.0->ultralytics>=8.0.167) (3.4)
Requirement already satisfied: urllib3<3,>=1.21.1 in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from requests>=2.23.0->ultralytics>=8.0.167) (1.26.16)
Requirement already satisfied: charset-normalizer<4,>=2 in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from requests>=2.23.0->ultralytics>=8.0.167) (3.2.0)
Requirement already satisfied: certifi>=2017.4.17 in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from requests>=2.23.0->ultralytics>=8.0.167) (2023.7.22)
Requirement already satisfied: typing-extensions in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from torch>=1.8.0->ultralytics>=8.0.167) (4.7.1)
Requirement already satisfied: networkx in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from torch>=1.8.0->ultralytics>=8.0.167) (3.1)
Requirement already satisfied: filelock in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from torch>=1.8.0->ultralytics>=8.0.167) (3.12.3)
Requirement already satisfied: sympy in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from torch>=1.8.0->ultralytics>=8.0.167) (1.12)
Requirement already satisfied: jinja2 in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from torch>=1.8.0->ultralytics>=8.0.167) (3.1.2)
Requirement already satisfied: colorama in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from tqdm>=4.64.0->ultralytics>=8.0.167) (0.4.6)
Requirement already satisfied: six>=1.5 in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from python-dateutil>=2.7->matplotlib>=3.2.2->ultralytics>=8.0.167) (1.16.0)
Requirement already satisfied: MarkupSafe>=2.0 in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from jinja2->torch>=1.8.0->ultralytics>=8.0.167) (2.1.3)
Requirement already satisfied: mpmath>=0.19 in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from sympy->torch>=1.8.0->ultralytics>=8.0.167) (1.3.0)
Installing collected packages: py-cpuinfo, seaborn, ultralytics
Successfully installed py-cpuinfo-9.0.0 seaborn-0.12.2 ultralytics-8.0.169
Collecting mediapipe>=0.10.3
  Using cached mediapipe-0.10.3-cp310-cp310-win_amd64.whl (50.2 MB)
Requirement already satisfied: absl-py in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from mediapipe>=0.10.3) (1.4.0)
Collecting sounddevice>=0.4.4
  Using cached sounddevice-0.4.6-py3-none-win_amd64.whl (199 kB)
Collecting flatbuffers>=2.0
  Using cached flatbuffers-23.5.26-py2.py3-none-any.whl (26 kB)
Requirement already satisfied: protobuf<4,>=3.11 in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from mediapipe>=0.10.3) (3.20.0)
Requirement already satisfied: attrs>=19.1.0 in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from mediapipe>=0.10.3) (23.1.0)
Requirement already satisfied: matplotlib in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from mediapipe>=0.10.3) (3.7.2)
Requirement already satisfied: numpy in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from mediapipe>=0.10.3) (1.23.5)
Collecting opencv-contrib-python
  Using cached opencv_contrib_python-4.8.0.76-cp37-abi3-win_amd64.whl (44.8 MB)
Collecting CFFI>=1.0
  Using cached cffi-1.15.1-cp310-cp310-win_amd64.whl (179 kB)
Requirement already satisfied: fonttools>=4.22.0 in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from matplotlib->mediapipe>=0.10.3) (4.42.1)
Requirement already satisfied: cycler>=0.10 in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from matplotlib->mediapipe>=0.10.3) (0.11.0)
Requirement already satisfied: python-dateutil>=2.7 in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from matplotlib->mediapipe>=0.10.3) (2.8.2)
Requirement already satisfied: contourpy>=1.0.1 in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from matplotlib->mediapipe>=0.10.3) (1.1.0)
Requirement already satisfied: pyparsing<3.1,>=2.3.1 in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from matplotlib->mediapipe>=0.10.3) (3.0.9)
Requirement already satisfied: pillow>=6.2.0 in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from matplotlib->mediapipe>=0.10.3) (9.5.0)
Requirement already satisfied: kiwisolver>=1.0.1 in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from matplotlib->mediapipe>=0.10.3) (1.4.5)
Requirement already satisfied: packaging>=20.0 in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from matplotlib->mediapipe>=0.10.3) (23.1)
Collecting pycparser
  Using cached pycparser-2.21-py2.py3-none-any.whl (118 kB)
Requirement already satisfied: six>=1.5 in b:\stablediffusionamd\stable-diffusion-webui-directml\venv\lib\site-packages (from python-dateutil>=2.7->matplotlib->mediapipe>=0.10.3) (1.16.0)
Installing collected packages: flatbuffers, pycparser, opencv-contrib-python, CFFI, sounddevice, mediapipe
Successfully installed CFFI-1.15.1 flatbuffers-23.5.26 mediapipe-0.10.3 opencv-contrib-python-4.8.0.76 pycparser-2.21 sounddevice-0.4.6
Collecting rich>=13.0.0
  Using cached rich-13.5.2-py3-none-any.whl (239 kB)
Collecting pygments<3.0.0,>=2.13.0
  Using cached Pygments-2.16.1-py3-none-any.whl (1.2 MB)
Collecting markdown-it-py>=2.2.0
  Using cached markdown_it_py-3.0.0-py3-none-any.whl (87 kB)
Collecting mdurl~=0.1
  Using cached mdurl-0.1.2-py3-none-any.whl (10.0 kB)
Installing collected packages: pygments, mdurl, markdown-it-py, rich
Successfully installed markdown-it-py-3.0.0 mdurl-0.1.2 pygments-2.16.1 rich-13.5.2
Launching Web UI with arguments: --medvram --precision full --no-half --no-half-vae --opt-split-attention --disable-nan-check --autolaunch --api
no module 'xformers'. Processing without...
no module 'xformers'. Processing without...
No module 'xformers'. Proceeding without it.
Civitai Helper: Get Custom Model Folder
Civitai Helper: Load setting from: B:\StableDiffusionAMD\stable-diffusion-webui-directml\extensions\Stable-Diffusion-Webui-Civitai-Helper\setting.json
Civitai Helper: No setting file, use default
[-] ADetailer initialized. version: 23.9.1, num models: 9
fatal: No names found, cannot describe anything.
Loading weights [8145104977] from B:\StableDiffusionAMD\stable-diffusion-webui-directml\models\Stable-diffusion\42424\meinahentai_v4.safetensors
Creating model from config: B:\StableDiffusionAMD\stable-diffusion-webui-directml\configs\v1-inference.yaml
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
Startup time: 188.3s (prepare environment: 175.4s, import torch: 4.1s, import gradio: 1.6s, setup paths: 1.5s, initialize shared: 1.7s, other imports: 0.3s, load scripts: 2.7s, create ui: 0.7s, gradio launch: 0.3s).
Loading VAE weights specified in settings: B:\StableDiffusionAMD\stable-diffusion-webui-directml\models\VAE\kl-f8-anime2.ckpt
Applying attention optimization: Doggettx... done.
Model loaded in 3.1s (load weights from disk: 0.5s, create model: 0.5s, apply weights to model: 0.8s, apply float(): 0.6s, load VAE: 0.3s, calculate empty prompt: 0.5s).
100%|██████████████████████████████████████████████████████████████████████████████████| 20/20 [00:30<00:00,  1.53s/it]
Total progress: 100%|██████████████████████████████████████████████████████████████████| 20/20 [00:20<00:00,  1.01s/it]
0: 640x448 1 face, 91.3ms
Speed: 3.0ms preprocess, 91.3ms inference, 1.0ms postprocess per image at shape (1, 3, 640, 448)
100%|████████████████████████████████████████████████████████████████████████████████████| 9/9 [00:09<00:00,  1.07s/it]
Total progress: 100%|██████████████████████████████████████████████████████████████████| 20/20 [00:35<00:00,  1.78s/it]
T
```

### Additional information

this didn't happen before the update

**Comment by matthewdeacon74** [comment]
I have experienced exactly the same issue.  Using Chrome on Windows 10, Radeon rx6600 8GB, driver v. 23.5.2,  
Always worked fine up through Thursday 8/31, but started locking up the app after I pulled latest on Saturday 9/2.  Now I can usually do 1 or 2 batches before it freezes, but sometimes it freezes on the first try.

**Comment by revons1984** [comment]
Same issue using Firefox on Windows 10. Radeon 7900xtx driver v. 23.82 but I also have this issue with In painting.

**Comment by RobotRunner** [comment]
this update stinks man! everything was working fine for me until a couple days ago when i couldn't generate a single image without my PC shutting down.
I updated by doing "git pull" in the directory through CMD, everything looks different and I can generate images now, but I can't even do batches of 4 anymore, also if I try to generate another image after doing 1, it just sticks on "waiting"

**Comment by Moonsome** [comment]
This is happening to me as well on both Firefox and Chrome, Radeon RX 6700 XT driver v. 23.7.2, Windows 10. Also happens when I use hi res fix in txt2img. Like everyone else, it worked fine before the update!

**Comment by Jorolesvaldo** [comment]
Same is happening to me. Sadly i've done a git pull today, 09/04 and suddenly inpainting stopped working. I'm on a RX 6600m 8GB, using the Brave Browser. Well, hope devs fix this as soon as they can, or at least roll back this update until things are better. But it's not anyone's fault! Love you devs, and thanks for all the support!

**Comment by matthewdeacon74** [comment]
Jorolesvaldo brings up a good point - I came here and complained about a new issue, but probably have never stated before that I am REALLY, TRULY APPRECIATIVE of all the hard work done here to open up SD to people without nVidia cards.  It's all basically voodoo to me, so I and hugely grateful that people who understand it are putting in the time to extend and expand it where I could never even comprehend doing it myself.
You folks are amazing, and your work here has been incredible.  Thanks for all you have done and will continue to do!

**Comment by Kestral5003** [comment]
> Jorolesvaldo brings up a good point - I came here and complained about a new issue, but probably have never stated before that I am REALLY, TRULY APPRECIATIVE of all the hard work done here to open up SD to people without nVidia cards. It's all basically voodoo to me, so I am hugely grateful that people who understand it are putting in the time to extend and expand it where I could never even comprehend doing it myself. You folks are amazing, and your work here has been incredible. Thanks for all you have done and will continue to do!

Very much agree, i can happily wait for a patch or an update, but i'd rather point an issue out so it can get fixed than wait on the off chance

**Comment by cheremo** [comment]
+1 here. using the 7900xt

what I can contribute is, that img2img and inpaint cause a complete freeze of SD  (not the PC's OS) in case I use any DPM++ Sampler. It's freezing instant after clicking "generate" or after the first rendered image. Other samplers like euler a dont cause a freeze.

--
what also surprised me today: 
I rendered some pictures with txt2img. Everything was fine, went afk. After I came back from AFK (2 hrs) the dedicated GPU's memory (20GB) was full (looked up at the taskmanager). I closed the SD instance and the dedicated GPU's memory dropped down to a normal idle level. Is this a normal behavior, that the GPUs memory stick at 100%?

**Comment by cheremo** [comment]
Now inpainting stopped working completely. The only think I get is a blurred or unchanged output.

**Comment by l0ggik** [comment]
Have the same problem with this and i can also contribute my observations.

For me this happens if i use the exact same prompt two times in a row. I started with imgtoimg, changed to texttoimg with the same prompt and it also freezes (and of course imgtoimg also freezes). Tried to use imgtoimg by just changing a single word back and forth every time i started a new generation and everything worked just fine.

**Comment by cheremo** [comment]
i fixed it for now by disabling a extensions (canvas-zoom, sd-webui-reactor, sd-webui-inpaint-anything, sd-webui-controlnet, sd-webui-segment-anything). Don't know which one is the bad guy

**Comment by d33pth0ug4t** [comment]
Good spot l0ggik!
Same problem here... blah blah... BUT (thanks mate) disabled "Persistent cond cache" and (so far) no problems!

**Comment by hookechoes** [comment]
Also having a similar problem.  When I use the built in upscaler my program freezes and crashes after just 1 or 2 images.  Working fine until last week.  On a 7900xt.

Really do appreciate the dev(s) for this branch though.  It's been fantastic up until this last update.

**Comment by cheremo** [comment]
Is it possible to pull not the latest version?

**Comment by hellridergit** [comment]
Same exact issue here, fresh SD installed from this repo 3 days ago.

> 

Oh. My. GOOOSH! It's been driving my nuts, I can finally inpaint multiple times in a row without having to restart SD completely. Thanks!

**Comment by CarsonChinachan** [comment]
I already have problem. I trying other sampling methods, some methods (euler a, euler, lms...) will work normal, some (dpm++2m karras, dpm++sde karras...) will stuck. I think it's because of the new version 1.6.0 of webui. 
but.... i dont how downgrade version base on directml:))

**Comment by lshqqytiger** [comment]
May be fixed in 92849df26f73b416d396b95c3fb8c64070fe3ad8.
